### ISSUE TITLE
General future of episode 7

### ISSUE THREAD

**Issue by klieret** [hsf-training/hsf-training-matplotlib#112]
Sorry if this is provocative (and I didn’t have time to go through it in detail): But what is episode 7 “Dimuon spectrum” actually teaching? It seems to me that there is a lot of code and an added dependency on another dataset, but no new matplotlib/mplhep features are actually being introduced. 

I understand that it's an interesting thing to do, but a similar example is also used in other tutorials (scikit-hep?) and this module should focus on matplotlib.

**Comment by amanmdesai** [comment]
I think the idea for episode 7 is to have a challenge (with solutions) for the students? 

https://hsf-training.github.io/hsf-training-matplotlib/07-dimuonspectrum/index.html#:~:text=the%20CMS%20experiment.-,Challenge,-!

**Comment by amanmdesai** [comment]
Perhaps if we convert this into a challenge episode completely?

**Comment by klieret** [comment]
I just feel that the difficulty in the exercise lies anywhere but in plotting. The plotting steps seem almost trivial, but the rest definitely is not for someone at the level where they aren't even familiar with matplotlib.

**Comment by amanmdesai** [comment]
yes I indeed agree with you. Maybe give python function to load dataset and other non-trivial stuff. and then the exercises only focuses on plotting.
